domains/auth/Auth.API/API/Controllers/TokenController.cs

```diff
@@ -0,0 +1,49 @@
+using System.ComponentModel.DataAnnotations;
+using API.Configuration;
+using API.Models;
+using API.Services;
+using API.Services.OidcProviders;
+using API.TokenStorage;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+
+namespace API.Controllers;
+
+
```

Comment:
Redundant newline

Reply:
Could this be fixed within the editorconfig?